arctic-report-card/ARCRC.ttl

```diff
@@ -232,508 +262,647 @@ Find current and past assessments of the Arctic Report Card Vital Signs by visit
+                   dc:date "2024-07-30T19:43:23Z"^^xsd:dateTime ;
+                   rdfs:label "Subsea Permafrost Indicator"@en ;
+                   skos:broadMatch "http://www.purl.dataone.org/odo/ECSO_00010130" .
+
+
+###  http://purl.dataone.org/odo/ARCRC_00000016
+odo:ARCRC_00000016 rdf:type owl:Class ;
+                   rdfs:subClassOf odo:ARCRC_00000513 ;
+                   obo:IAO_0000115 "Applying and centering Indigenous perspectives and observations of Arctic change" ;
+                   dc:creator <http://orcid.org/0009-0007-8384-5074> ;
+                   dc:date "2024-07-30T19:43:23Z"^^xsd:dateTime ;
+                   rdfs:label "Working with Communities to Observe the Arctic"@en .
+
+
+###  http://purl.dataone.org/odo/ARCRC_00000017
+odo:ARCRC_00000017 rdf:type owl:Class ;
+                   rdfs:subClassOf odo:ARCRC_00000513 ;
+                   obo:IAO_0000115 "Here, we refer to “western Alaska” salmon as those spawning in watersheds that drain into the eastern Bering Sea." ;
+                   dc:creator <http://orcid.org/0009-0007-8384-5074> ;
+                   dc:date "2024-07-30T19:43:23Z"^^xsd:dateTime ;
+                   rdfs:label "Western Alaska Salmon"@en ;
+                   skos:relatedMatch "http://purl.dataone.org/odo/SALMON_00000215" .
+
+
+###  http://purl.dataone.org/odo/ARCRC_00000018
+odo:ARCRC_00000018 rdf:type owl:Class ;
+                   rdfs:subClassOf odo:ARCRC_00000040 ;
+                   obo:IAO_0000115 "Any product of the condensation of atmospheric water vapor that falls under gravity" ;
+                   dc:creator <http://orcid.org/0009-0007-8384-5074> ;
+                   dc:date "2024-07-30T19:43:23Z"^^xsd:dateTime ;
+                   dc:description "This Key Variable is used to inform Vital Sign, 'Precipitation.'" ;
+                   rdfs:label "precipitation"@en ;
+                   owl:sameAs "http://purl.obolibrary.org/obo/ENVO_01000703" .
+
+
+###  http://purl.dataone.org/odo/ARCRC_00000019
+odo:ARCRC_00000019 rdf:type owl:Class ;
+                   rdfs:subClassOf odo:ARCRC_00000040 ;
+                   obo:IAO_0000115 "Estimated total amount of ice in the sediment column and represents total ice content" ;
+                   dc:creator <http://orcid.org/0009-0007-8384-5074> ;
+                   dc:date "2024-07-30T19:43:23Z"^^xsd:dateTime ;
+                   dc:description "This Key Variable is used to inform Other Indicator, 'Subsea Permafrost'" ;
+                   rdfs:label "total ice content"@en ;
+                   skos:relatedMatch "http://purl.dataone.org/odo/ECSO_00002173" .
+
+
+###  http://purl.dataone.org/odo/ARCRC_00000020
+odo:ARCRC_00000020 rdf:type owl:Class ;
+                   rdfs:subClassOf odo:ARCRC_00000040 ;
+                   obo:IAO_0000115 "Long-term documentation of weather, ocean, sea ice, and landscape conditions and context to wildlife, harvesting, and cultural and community activities" ;
```

Comment:
Can you provide a source for this definition? There are multiple definitions of `Indigenous observation` and we should be clear which one this pertains to.

Reply:
I just got this definition from the essay itself. Is there a way to add a link to the essay in the definition? Or should I look for an outside definition in an existing ontology?

Reply:
Yes, to cite the source for a definition, use http://purl.obolibrary.org/obo/IAO_0000119, which is an annotation property for "definition source". There are other examples of its use in the ontology, but you could use something like:

```
odo:ARCRC_00000020 rdf:type owl:Class ;
                   obo:IAO_0000119 "Indigenous observation as defined in https://arctic.noaa.gov/Report-Card/..." .
```

It also looks like there is still a prefix mistake in the ontology for `obo:IAO_0000119` whic his mistakenly referred to as `odo:IAO_0000119` in the `odo` prefix. This needs to be fixed in several places in the ontology.

Reply:
Added in definition source with a link to the ARCRC Essay 2023
`obo:obo:IAO_0000119 "Indigenous observations as defined in the Arctic Report Card 2023 Essay at https://arctic.noaa.gov/report-card/report-card-2023/nunaaqqit-savaqatigivlugich-working-with-communities-to-observe-the-arctic/"`